streams/tr.m3u

```diff
@@ -42,6 +42,8 @@ https://cdn-bursatv.yayin.com.tr/bursatv/bursatv/playlist.m3u8
 https://win1.yayin.com.tr/bursatv/bursatv/playlist.m3u8
 #EXTINF:-1 tvg-id="CanTV.tr" status="online",Can TV (720p)
 http://canbroadcast.com:7000/canlican/tv.m3u8
+#EXTINF:-1 tvg-id="CartoonNetworkTurkey.tr" status="timeout",Cartoon Network Turkey
+https://ocanli1.etvserver.com/live_sd/cartoonnetwork/chunks.m3u8
```

Comment:
I have https://59cba4d34b678.streamlock.net/live2/cartoon-network.stream/playlist.m3u8?hash=2c91747fa735d98851357ac77aed0e8d for this one.

Reply:
Both links don't work

Reply:
> Both links don't work

YEP